app/src/main/java/com/infomaniak/swisstransfer/ui/components/transfer/TransferItemList.kt

```diff
@@ -53,37 +60,54 @@ fun TransferItemList(
     }
 
     LazyColumn(
-        verticalArrangement = Arrangement.spacedBy(Margin.Medium),
+        verticalArrangement = Arrangement.spacedBy(Margin.Mini),
         contentPadding = contentPadding,
     ) {
 
         item { SmallWindowScreenTitle(title = stringResource(titleRes)) }
 
-        items(
-            count = getTransfers().count(),
-            key = { getTransfers()[it].uuid },
-            contentType = { getTransfers()[it] },
-            itemContent = {
-                val transfer = getTransfers()[it]
-                SwipeToDismissComponent(
-                    contentShape = itemShape,
-                    onSwiped = { onSwiped(transfer.uuid) },
+        getTransfers().forEach { (section, transfers) ->
+            stickyHeader {
+                Box(
+                    modifier = Modifier
+                        .fillParentMaxWidth()
+                        .background(SwissTransferTheme.materialColors.background)
```

Comment:
This background is really needed ?

Reply:
Yes, this way the sticky header is a whole width background-colored rectangle with text on it. Else the sticky header is only text on a transparent background